python/KMS_IMPORTED_KEY_EXPIRATION_CHECK/KMS_IMPORTED_KEY_EXPIRATION_CHECK.py

```diff
@@ -0,0 +1,437 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+'''
+#####################################
+##            Gherkin              ##
+#####################################
+
+Rule Name:
+  KMS_IMPORTED_KEY_EXPIRATION_CHECK
+
+Description:
+  Check that KMS imported keys are with or without expiration. This rule is NON_COMPLAINT if the imported key has expiration set and it is less than or equal to the daysToExpiration parameter.

```

Comment:
Change to
`Checks whether KMS imported keys are marked for expiration within the specified number of days. This rule is NON_COMPLAINT if the imported key has expiration set and it is less than or equal to the daysToExpiration parameter (default: 14 days).`